### DISCUSSION TITLE
Please help me for Anime collection

### DISCUSSION THREAD

**Discussion by aptx48691412** [NAIST-SE/AttractingContributors#19] [General]
Please help me with my anime list.
https://github.com/aptx48691412/AttractingContributors-Anime

**Comment by wowry** [comment]
Added!

**Comment by tkmaeda** [comment]
Sent you a PR!

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors

**Comment by ElefunAi** [comment]
I have added Anime list!
Please check and help me!

https://github.com/ElefunAi/AttractingContributors-texteditor

**Comment by t-matsumoto4758** [comment]
Added!
please help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by tktkbohshi** [comment]
added!
plz help me...
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga
